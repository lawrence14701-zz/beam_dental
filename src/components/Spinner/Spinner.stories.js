import React from "react"
import Spinner from "./Spinner"
import { ThemeProvider } from "@material-ui/styles"
import muiTheme from "../../Theme/theme"

export default {
  title: "Spinner",
  component: Spinner,
  decorators: [
    (Story) => (
      <ThemeProvider theme={muiTheme}>
        <Story />
      </ThemeProvider>
    ),
  ],
}

const Template = (args) => <Spinner {...args} />

export const Primary = Template.bind({})
