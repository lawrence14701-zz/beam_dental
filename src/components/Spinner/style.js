import { makeStyles } from '@material-ui/core/styles';

const useStyles = ((theme) => ({
    root: {
        position: 'absolute',
        top:" 50%",
        left: '50%',
    }
}));

export default makeStyles(useStyles)