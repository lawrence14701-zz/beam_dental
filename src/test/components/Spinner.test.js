import React from "react"
import Spinner from "../../components/Spinner/Spinner"
import "../jestsetup"

const Composition = () => <Spinner />

describe("<Spinner />", () => {
  it("should render a Spinner", () => {
    const wrapper = shallow(<Composition />)
    expect(wrapper.find(Spinner)).toHaveLength(1)
  })
  test("renders a spinning component", () => {
    const wrapper = shallow(<Spinner />)
    expect(wrapper).toMatchSnapshot()
  })
})
