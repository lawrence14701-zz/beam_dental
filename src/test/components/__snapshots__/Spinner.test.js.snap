// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Composition /> renders a spinning component 1`] = `ShallowWrapper {}`;
